iidm/iidm-api/src/main/java/com/powsybl/iidm/network/util/Networks.java

```diff
@@ -33,8 +33,8 @@ public final class Networks {
     private Networks() {
     }
 
-    public static boolean isBusValid(int branchCount) {
-        return branchCount >= 1;
+    public static boolean isBusValid(int feederCount) {
```

Comment:
@zamarrenolm we have to choose between this fix, that seems to be close to the node/breaker voltage level condition to the reasonable one that is 
````
public static boolean isBusValid(int branchCount, int feederCount) {
    return branchCount >= 1 || branchCount == 0 && feederCount > 1;
}
````

Reply:
For me this condition is ok. Only isolated nodes with no equipment connected will be considered as invalid buses. Components with only one bus but that have at least one equipment connected to it will be considered.